Sources/MessagingPush/MessagingPushConfigOptions.swift

```diff
@@ -9,7 +9,6 @@
  pushMessagingConfig.autoTrackPushEvents = false
  ```
  */
-// TODO: [CDP] Update docs to after module initialization changes for clarity on its usage
```

Comment:
Did we complete this TODO too? 🤔

Reply:
We have a separate ticket for docs, ill add this in it.